# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/kasc/libsdl_perl/console_fix.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./SDL_perl.xs	2003-01-03 06:14:35.000000000 +0100
+++ ./SDL_perl.xs	2003-05-26 16:04:15.000000000 +0200
@@ -67,7 +67,7 @@
 
 #ifdef HAVE_SDL_CONSOLE
 #include "CON_console.h"
-#include "CON_consolecommands.h"
+/*#include "CON_consolecommands.h"*/
 #include "DT_drawtext.h"
 #endif
 
